app/services/TaxCalculationService.scala

```diff
@@ -72,13 +74,24 @@ class TaxCalculationService @Inject()(
             Logger.debug(s"Unexpected ${r.status} response getting tax calculation from tax-calculation-service")
             TaxCalculationUnexpectedResponse(r)
         },
-        onError = {
-          case e =>
-            Logger.debug(e.toString)
-            t.completeTimerAndIncrementFailedCounter()
-            Logger.warn("Error getting tax calculation from tax-calculation-service", e)
-            TaxCalculationErrorResponse(e)
+        onError = { e =>
+          Logger.debug(e.toString)
+          t.completeTimerAndIncrementFailedCounter()
+          Logger.warn("Error getting tax calculation from tax-calculation-service", e)
+          TaxCalculationErrorResponse(e)
         }
       )
     }
+
+  def getTaxYearReconciliations(nino: Nino, startYear: Int, endYear: Int)(
+    implicit headerCarrier: HeaderCarrier): Future[List[TaxYearReconciliation]] =
```

Comment:
Missing tests, do we want too track metrics on this similar to getTaxCalculation?

Reply:
Discussed this with you